10.Mender-server/docs.md

```diff
@@ -2166,6 +2166,90 @@ New changes in integration since 3.0.0:
     ...
 
 
+## Mender 3.0.1
+
+_Release date 09.29.2021_
+
+### Statistics
+
+A total of 585 lines added, 265 removed (delta 320)
+
+| Developers with the most changesets | |
+|---|---|
+| Manuel Zedel | 9 (42.9%) |
+| Fabio Tranchitella | 6 (28.6%) |
+| Kristian Amlie | 4 (19.0%) |
+| Ole Petter Orhagen | 1 (4.8%) |
+| Alf-Rune Siqveland | 1 (4.8%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Fabio Tranchitella | 437 (72.2%) |
+| Alf-Rune Siqveland | 55 (9.1%) |
+| Manuel Zedel | 51 (8.4%) |
+| Ole Petter Orhagen | 43 (7.1%) |
+| Kristian Amlie | 19 (3.1%) |
+
+| Developers with the most lines removed | |
+|---|---|
+| Kristian Amlie | 3 (1.1%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 21 (100.0%) |
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 605 (100.0%) |
+
+| Employers with the most hackers (total 5) | |
+|---|---|
+| Northern.tech | 5 (100.0%) |
+
+### Changelogs
+
+#### deployments-enterprise (3.0.1)
+
+New changes in deployments-enterprise since 3.0.0:
+
+* add update control map eevaluation
```

Comment:
Typo: evaluation.

When the errors are these obvious, it is okey to edit the final text.